.github/workflows/job_python_unit_tests.yml

```diff
@@ -121,7 +121,7 @@ jobs:
       #
 
       - name: Python API Tests
-        # if: ${{ fromJSON(inputs.affected-components).Python_API.test && runner.arch != 'ARM64' }} # Ticket: 126380, 127101
+        if: ${{ fromJSON(inputs.affected-components).Python_API.test && runner.arch != 'ARM64' }} # Ticket: 126380
```

Comment:
don't we want to run it on ARM?

Reply:
We do, but there was an issue on ARM: 126380

Reply:
Ok, they doesn't reproduce anyway